### ISSUE TITLE
Request for training losses

### ISSUE THREAD

**Issue by Brianwind** [starVLA/starVLA#99]
Hi, thank you for your great work

I can't open links you provided here: https://github.com/starVLA/starVLA/issues/68
Can you share the loss details in other ways? I am particularly interested in the absolute loss values at final convergence, as well as the overall convergence trend, especially about florence-2 + gr00t.

Thank you for your time and for open-sourcing this project!

**Comment by JinhuiYE** [comment]
Here U are. https://wandb.ai/jinhuiye/starVLA_simplerEnv?nw=nwuserjinhuiye

<img width="2878" height="1454" alt="Image" src="https://github.com/user-attachments/assets/d106850b-8676-4524-a5df-bccb1f9356e0" />
